includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -293,7 +293,7 @@ protected function layered_nav_dropdown( $terms, $taxonomy, $query_type ) {
 				if ( jQuery().selectWoo ) {
 					var wc_layered_nav_select = function() {
 						jQuery( '.dropdown_layered_nav_" . esc_js( $taxonomy_filter_name ) . "' ).selectWoo( {
-							placeholder: '" . esc_js( $any_label ) . "',
+							placeholder: '" . wp_specialchars_decode( esc_js( $any_label ), ENT_QUOTES, get_bloginfo( 'charset' ) ) . "',
```

Comment:
Shouldn't this be
`placeholder: '" . esc_js( wp_specialchars_decode( $any_label, ENT_QUOTES, get_bloginfo( 'charset' ) ) ). "'`
instead?

Reply:
Example attribute label I am concerned about:
`Test'.(alert('Hello World!'),' label').'`

Reply:
Unfortunately esc_js will convert the double quotes again, that is why I added it inside. Will do some more testing and see what I can achieve without opening it up to an XSS attack.